db/migrate/20190321195704_create_procedure_instances.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProcedureInstances < ActiveRecord::Migration[5.2]
+  def change
+    create_table :procedure_instances do |t|
+      t.integer :procedure_id
```

Comment:
We did not cover foreign_keys in class but this is a good place to use them. I would change this to `t.references :procedure, foreign_key: true`. This will create the column `procedure_id` and makes sure that a procedure with id `procedure_id` exists in the database.